bgstally/state.py

```diff
@@ -34,6 +34,7 @@ def load(self):
         self.DiscordPostStyle:tk.StringVar = tk.StringVar(value=config.get_str('XDiscordPostStyle', default=DiscordPostStyle.EMBED))
         self.DiscordActivity:tk.StringVar = tk.StringVar(value=config.get_str('XDiscordActivity', default=DiscordActivity.BOTH))
         self.EnableOverlay:tk.StringVar = tk.StringVar(value=config.get_str('XEnableOverlay', default=CheckStates.STATE_ON))
+        self.EnableOverlayCurrentTick:tk.StringVar = tk.StringVar(value=config.get_str('XEnableOverlayCurrentTick', default=CheckStates.STATE_ON))
```

Comment:
Could you rename this setting to `BGST_EnableOverlayCurrentTick` throughout the code - The old 'X' prefix was from Tez's original version and isn't a very good unique prefix identifier.